lnbits/extensions/cashu/tasks.py

```diff
@@ -28,6 +28,10 @@ async def wait_for_paid_invoices():
 
 
 async def on_invoice_paid(payment: Payment) -> None:
-    if payment.extra and not payment.extra.get("tag") == "cashu":
+    if not payment.extra:
```

Comment:
This is where the logic was changed, not just a simple refactor. I think the suggested behaviour is more correct than the old one.

Reply:
we could also try to evalute if there is a good reason for payment.extra to be Optional, maybe we can get rid of it altogether?